cicd/migration_helpers.py

```diff
@@ -110,13 +112,21 @@ def download_ibsolution(ib_host, api_token, solution_path, write_to_local=True):
   """
 
   # TODO: Check if file exists first
-  # Read in .ibsolution file
   resp = read_file_through_api(ib_host, api_token, solution_path)
 
   if write_to_local:
-    with open(solution_path.split('/')[-1], 'wb') as fd:
+    solution_name = solution_path.split('/')[-1]
+    with open(solution_name, 'wb') as fd:
       fd.write(resp.content)
 
+    if unzip_solution:
+      zip_path = solution_name.replace('ibsolution', 'zip')
```

Comment:
```suggestion
      zip_path = Path(solution_path).with_suffix(".zip").name
```

Probably better to do `_solution_path = Path(solution_path)` at the top so that we don't need to keep recreating it